src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs

```diff
@@ -1481,6 +1481,46 @@ public class C
             await VerifyCSharpAdditionalFileFixAsync(source, shippedText, unshippedText, fixedUnshippedText);
         }
 
+        [Theory]
+        [WorkItem(4749, "https://github.com/dotnet/roslyn-analyzers/issues/4749")]
+        [InlineData("\r\n")] // Windows line ending.
+        [InlineData("\n")] // Linux line ending.
+        [InlineData("\r")] // macOS line ending.
```

Comment:
This was only the Mac line ending prior to OSX. There is no need to support it.

```suggestion
```

Reply:
Mildly interesting side note: the first release of OSX predated .NET Framework 1.0 by almost a year, so C# code should really never need to handle these.